{{ cookiecutter.project_slug }}/api/{{ cookiecutter.project_slug }}/apps/accounts/api/v1/serializers/password.py

```diff
@@ -45,7 +45,7 @@ def save(self, **kwargs):
 
 class ConfirmResetPasswordSerializer(serializers.Serializer):
     password = serializers.CharField(max_length=128, write_only=True, style={"input_type": "password"})
-    signature = serializers.CharField(max_length=71, write_only=True)
+    signature = serializers.CharField(max_length=128, write_only=True)
```

Comment:
Exact value for current configuration is `87` but then we will have no margin. I as I understand length of `signature` can be different if we change algorithm so I decide give it some margin.